pennylane/qchem/vibrational/vibrational_class.py

```diff
@@ -125,3 +125,93 @@ def optimize_geometry(molecule, method="rhf"):
 
     scf_result = _single_point(mol_eq, method)
     return mol_eq, scf_result
+
+
+def _get_rhf_dipole(scf_result):
+    """
+    Given an restricted Hartree-Fock object, evaluate the dipole moment
```

Comment:
```suggestion
    Given a restricted Hartree-Fock object, evaluate the dipole moment
```

Reply:
Could rephrase to "Evaluate the dipole moment in the restricted Hartree-Fock state for a restricted Hartree-Fock object"